packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -144,7 +146,7 @@ export default class AutomateProvider extends GenericProvider {
         sha: tileData.split('-')[0] // drop build id
       }));
     }
-    return { tiles: tiles, domInfoSha: tileResponse.dom_sha };
+    return { tiles: tiles, domInfoSha: tileResponse.dom_sha, windowHeight: windowHeight * dpr };
```

Comment:
`windowHeight * dpr` can be floating point correct ?
I understand that we are doing `to_i` in API, but code in CLI can be misrepresenting that API accepts floating point number.
Hence, I would suggest to make this int instead of float.

Reply:
Then that should be done at all the places. floating digit comes for mobile as of now.

Reply:
Ack I would add ceil function instead as we have seen height gets ceiled in images.